compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -231,6 +231,10 @@ impl CodegenBackend for CraneliftCodegenBackend {
         }
     }
 
+    fn print_version(&self) {
+        println!("cranelift version: {}", cranelift_codegen::VERSION);
```

Comment:
I don't have a strong preference either way, but maybe
```suggestion
        println!("Cranelift version: {}", cranelift_codegen::VERSION);
```